docs/proposals/organizations.md

```diff
@@ -0,0 +1,125 @@
+# Organizations
+
+## Problems
+ 1.  We need to be able to allocate quota to a group of projects.
```

Comment:
Cluster administrators want to allocate a pool of resources to a subset of their company, and delegate the authority to manage how that pool is assigned to an organization administrator.